### ISSUE TITLE
[4.3.1] - Unable to determine the version of the Android plugin to include in the build scan, due to java.lang.ClassNotFoundException: com.android.builder.Version

### ISSUE THREAD

**Issue by jaredsburrows** [gradle/gradle#3475]
## Description
Related: https://github.com/KeepSafe/dexcount-gradle-plugin/issues/224
Google moved this class from `com.android.builder.Version` to `com.android.builder.model.Version`.

## Exception
```
Unable to determine the version of the Android plugin to include in the build scan, due to java.lang.ClassNotFoundException: com.android.builder.Version
```

**Comment by oehme** [comment]
@ldaley Could you look into this?

**Comment by jaredsburrows** [comment]
@oehme Is there a reason by build scan needs the `Version` class?

**Comment by bmuschko** [comment]
@jaredsburrows Can you provide us with the stack trace for the message you provided above?

**Comment by oehme** [comment]
I don't think we need a stacktrace here, we know what the problem is already.

The scan UI displays the Android plugin version, so we get it through that class. Should be a rather simple fix.

**Comment by jaredsburrows** [comment]
We can either check of the instance of the plugins at runtime or do something like this: https://github.com/Kotlin/dokka/blob/master/runners/gradle-plugin/src/main/kotlin/main.kt#L30.

**Comment by etiennestuder** [comment]
Build scan plugin version 1.10.2 (already released) properly handles the reported scenario.

**Comment by oehme** [comment]
Thanks @etiennestuder!

**Comment by etiennestuder** [comment]
@oehme My bad. It is fixed on master but not in 1.10.2 :-(

**Comment by jaredsburrows** [comment]
@etiennestuder When will you open source the scan plugin? I will close this since you said it is already fixed.

**Comment by etiennestuder** [comment]
@jaredsburrows We have no plans to open-source the b

**Comment by jaredsburrows** [comment]
@etiennestuder Why open this if you said it is fixed?

**Comment by etiennestuder** [comment]
@jaredsburrows Because the fix is not yet released.

**Comment by etiennestuder** [comment]
This is fixed in the now release build scan plugin v1.10.3.

**Comment by jaredsburrows** [comment]
Thank you!
